### PR TITLE
[Fix] not used notify when creating a new case (cause it's still a draft) & Separeted API notification

### PR DESCRIPTION
**Pull request by khihadysucahyo**
## Overview
1. not used notify when creating a new case (cause it's still a draft)
2. Separeted API notification

### CODE REVIEW

File: api/notifications/package.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "name": "notifications",
+    "versopm": "1.0.0"
```
Comment (firmanJS) [inline api/notifications/package.json:3]: Ini sengaja atau typo ?

**Review by firmanJS — APPROVED** [review]

File: api/notifications/package.json (hunk 1 above)
Comment (khihadysucahyo) [reply api/notifications/package.json]: wkwk typo atuh kang

**Comment by khihadysucahyo** [comment]
sundul kang @sandisunandar99

**Comment by sandisunandar99** [comment]
sory oy baru selesai ngisi kuliah ,,, wkwkwkkwkw

**Review by sandisunandar99 — APPROVED** [review]

**Pull request merged**